cmd/opampsupervisor/supervisor/supervisor.go

```diff
@@ -576,7 +700,7 @@ func (s *Supervisor) loadAgentEffectiveConfig() {
 		effectiveConfigBytes = s.composeExtraLocalConfig()
 	}
 
-	s.effectiveConfig.Store(string(effectiveConfigBytes))
+	s.mergedConfig.Store(string(effectiveConfigBytes))
```

Comment:
Good call here. I noticed that there are a lot of "misnomers" with this PR now (a lot of things refer to "effective config" instead of "merged config"). I went through and resolved all the ones I could find.

In this particular case, I removed the read from the file here; Interestingly, it seems we clobber whatever was in mergedConfig later on in this function anyways, so there is currently no reason to do this.